src/repl.ts

```diff
@@ -166,8 +166,8 @@ function repl(this: any, options: any) {
 
     // lookup repl by id, using steams to submit cmd and send back response
 
-    let replID = msg.id || options.host + ':' + options.port
-    let replInst = replMap[replID]
+    let replID = msg.id
+    let replInst = replID ? replMap[replID] : null
```

Comment:
Because the address is no longer enough to produce a meaningful fallback REPL ID - we must require the REPL ID argument.